packages/ekstep_data_pipelines/audio_analysis/analyse_speaker.py

```diff
@@ -8,14 +8,13 @@
 
 
 def analyse_speakers(
-    embed_file_path,
-    source,
-    min_cluster_size,
-    partial_set_size,
-    min_samples,
-    fit_noise_on_similarity,
+        embed_file_path,
+        source,
+        min_cluster_size,
+        partial_set_size,
+        min_samples,
+        fit_noise_on_similarity,
```

Comment:
We format the code using code formatting tool Black. If pylint is complaining about that then it is a conflict between the them.  Can you check if Black formatting is intact

command to format:
make format

Reply:
let me check using black.